ktrash is being renamed ktrash4 to allow kdebase-runtime be co-installable with KDE 3.
--- a/kioslave/trash/CMakeLists.txt
+++ b/kioslave/trash/CMakeLists.txt
@@ -39,11 +39,11 @@
 set(ktrash_SRCS ktrash.cpp )
 
 
-kde4_add_executable(ktrash NOGUI ${ktrash_SRCS})
+kde4_add_executable(ktrash4 NOGUI ${ktrash_SRCS})
 
-target_link_libraries(ktrash  ${KDE4_KIO_LIBS} )
+target_link_libraries(ktrash4  ${KDE4_KIO_LIBS} )
 
-install(TARGETS ktrash ${INSTALL_TARGETS_DEFAULT_ARGS})
+install(TARGETS ktrash4 ${INSTALL_TARGETS_DEFAULT_ARGS})
 
 ########### next target ###############
 
--- a/kioslave/trash/ktrash.cpp
+++ b/kioslave/trash/ktrash.cpp
@@ -28,17 +28,17 @@
 int main(int argc, char *argv[])
 {
     //KApplication::disableAutoDcopRegistration();
-    KCmdLineArgs::init( argc, argv, "ktrash", "kio_trash",
-                        ki18n( "ktrash" ),
+    KCmdLineArgs::init( argc, argv, "ktrash4", "kio_trash",
+                        ki18n( "ktrash4" ),
                         KDE_VERSION_STRING ,
                         ki18n( "Helper program to handle the KDE trash can\n"
-				   "Note: to move files to the trash, do not use ktrash, but \"kioclient move 'url' trash:/\"" ));
+				   "Note: to move files to the trash, do not use ktrash4, but \"kioclient move 'url' trash:/\"" ));
 
     KCmdLineOptions options;
     options.add("empty", ki18n( "Empty the contents of the trash" ));
     //{ "migrate", I18N_NOOP( "Migrate contents of old trash" ), 0 },
     options.add("restore <file>", ki18n( "Restore a trashed file to its original location" ));
-    // This hack is for the servicemenu on trash.desktop which uses Exec=ktrash -empty. %f is implied...
+    // This hack is for the servicemenu on trash.desktop which uses Exec=ktrash4 -empty. %f is implied...
     options.add("+[ignored]", ki18n( "Ignored" ));
     KCmdLineArgs::addCmdLineOptions( options );
     KApplication app;
